Buddies/Activities/View Activity/ViewActivityController.swift

```diff
@@ -66,11 +66,17 @@ class ViewActivityController: UIViewController {
     }
 
     @objc func onReportTap(_ sender: Any) {
-        guard self.curMemberStatus != .owner else { return }
-        showCancelableAlert(withMsg: "What's wrong with this activity?", withTitle: "Report Activity", withAction: "Report", showTextEntry: true) { (didConfirm, msg) in
-            guard didConfirm, let activity = self.curActivity else { return }
-            FirestoreManager.reportActivity(activity.activityId, reportMessage: msg ?? "")
-        }
+        guard self.curMemberStatus != .owner,
+            let parentNav = BuddiesStoryboard.Report.viewController() as? UINavigationController,
```

Comment:
I'm a little confused by the naming convention here - how is report the parent nav if we're presenting it?